R/inven.R

```diff
@@ -50,38 +50,38 @@ inven <- function(x,obj,need=NULL,crump=TRUE) {
 #' @export
 inventory <- function(x,obj,..., .strict = TRUE) {
   
-  need <- select_vars(names(param(x)), ...)
+  oname <- as.character(as.list(match.call())$obj)
+
+  need <- select_vars(names(param(x)),...)
   
-  if(!length(need)) {
-    return(x)
-  }
+  if(!length(need)) need <- names(param(x))
```

Comment:
@dpastoor I think this is needed to make the "default" (nothing in `...`) parameter requirement all of the parameters in the model.